databridge.examples/databridge.examples.kafka-jsonata-aas/pom.xml

```diff
@@ -10,7 +10,8 @@
 
 	<artifactId>databridge.examples.kafka-jsonata-aas</artifactId>
 	<name>Kafka jsonATA AAS</name>
-	<description>An Integration example using kafka as a data source, jsonATA as data transformer and AAS as sink</description>
+	<description>An Integration example using kafka as a data source, jsonATA as
+		data transformer and AAS as sink</description>
```

Comment:
Please revert cosmetic changes

Reply:
this also is due to the official basyx formatter, but you're right, here it really makes things ugly! Will be reverted with the next commit.